src/less/components/commands.less

```diff
@@ -11,10 +11,7 @@ header {
   justify-content: space-between;
   height: 50px;
   padding:
-    10px
-    calc(100vw - (env(titlebar-area-x, 0) + env(titlebar-area-width, 0)))
-    10px
-    env(titlebar-area-x, 0);
+    10px;
```

Comment:
I think because this is specifically for Windows/Linux styles, we'd want to make this styling different depending on OS.

 We can do this conditionally by applying a CSS class (e.g. `is-mac`) in the React code via `process.platform === 'darwin'` and applying different paddings depending on that class.

Reply:
Hey there! I have made recommended changes to the PR i.e. a CSS class that gets applied only for Mac OS. FYI  ```header .is-mac ``` selector in ```commands.less``` makes sure the padding specified with ```.is-mac``` styles gets higher specificity.